docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -195,4 +195,15 @@ Alternatively, navigate to your Data storage directory and enter the following c
 
 *Further Reading*: https://docs.couchbase.com/server/current/learn/data/data.html#metadata[Metadata], https://docs.couchbase.com/server/current/rest-api/rest-bucket-create.html#evictionpolicy[evictionPolicy].
 
+[#CB90061]
+=== Disk Write Queue (CB90061)
+
+*Background*: Having items in the disk write queue for a long time can backoffs.
```

Comment:
Bit more description, effects of backoffs what are they etc?

Reply:
I'm not exactly sure what backoffs are, do you have any ideas about what would be good to put here?

Reply:
not exactly sure either :p, worth some research in past tickets/JIRA?

Reply:
aiui backoffs in this case would usually be saying "stop / slow down sending data, we can't persist it to disk quickly enough". If there are many items in the queue, it could either be the physical storage can't keep up (DWQ is draining, just not quickly enough) or something seriously has gone wrong (DWQ not draining because storage is not responding or has high latency)

Reply:
Thank you! I'll add something in about that :))